### ISSUE TITLE
Installation via pip fails

### ISSUE THREAD

**Issue by gwpantazes** [bemeurer/beautysh#22]
Installing beautysh via pip fails, see failure output below.

```
$ pip install beautysh
Collecting beautysh
  Using cached beautysh-3.7.tar.gz
Installing collected packages: beautysh
  Running setup.py install for beautysh ... error
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/sv/5gh4tk990539nzhdt311pg0h000_fp/T/pip-build-9uWIcs/beautysh/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/sv/5gh4tk990539nzhdt311pg0h000_fp/T/pip-knmUwV-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    file beautysh.py (for module beautysh) not found
    creating build
    creating build/lib
    creating build/lib/beautysh
    copying beautysh/__init__.py -> build/lib/beautysh
    copying beautysh/__main__.py -> build/lib/beautysh
    copying beautysh/beautysh.py -> build/lib/beautysh
    file beautysh.py (for module beautysh) not found
    running install_lib
    creating /Library/Python/2.7/site-packages/beautysh
    error: could not create '/Library/Python/2.7/site-packages/beautysh': Permission denied
    
    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/sv/5gh4tk990539nzhdt311pg0h000_fp/T/pip-build-9uWIcs/beautysh/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/sv/5gh4tk990539nzhdt311pg0h000_fp/T/pip-knmUwV-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/sv/5gh4tk990539nzhdt311pg0h000_fp/T/pip-build-9uWIcs/beautysh/
```

Machine specs: 
MacOS Sierra 10.12.6
```sh
$ pip --version
pip 9.0.1 from /Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg (python 2.7)
```

**Comment by jwhonce** [comment]
@gwpantazes I don't have a mac to test against, but my guess as a Linux person is: /Library/Python/2.7/site-packages/beautysh is owned by root not a user. 

Does pip install --user work on macos?

**Comment by gwpantazes** [comment]
I've tried again, and I can't reproduce this anymore.

- Tried regular pip install again, and it worked. (installed for both Python2 and Python3)
- Uninstalled both via pip
- Reinstalled successfully as well.

In between these attempts, I think I ran a homebrew update which might have upped some versions here and there, including for Python. I'm not sure though, and I'm not finding an obvious way to print out a log of what homebrew changed. Here are the new pip versions, which I don't expect to help.
```
pip3 --version
pip 9.0.1 from /usr/local/lib/python3.6/site-packages (python 3.6)
pip2 --version
pip 9.0.1 from /usr/local/lib/python2.7/site-packages (python 2.7)
```
@jwhonce Should this be closed?
